core/src/main/scala/org/apache/spark/storage/memory/MemoryStore.scala

```diff
@@ -21,9 +21,14 @@ import java.io.OutputStream
 import java.nio.ByteBuffer
 import java.util.LinkedHashMap
 
+import scala.collection.JavaConverters._
 import scala.collection.mutable
 import scala.collection.mutable.ArrayBuffer
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
shall we create a dedicated thread pool (with a reasonable capacity) to run the closing tasks?